File: Lab2/C#/MyString.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.IO;
using System.Linq;

namespace Lab2
{
    public class MyString
    {
        private readonly char[] Value;

        public MyString(char[] value)
        {
            this.Value = value;
        }

        public char[] GetValue()
        {
            return this.Value;
        }
        
    }
}

File: Lab2/C++/myString.cpp
#include <cstring>
#include "myString.h"

using namespace std;


char* MyString::getValue() {
    return this->Value;
}

MyString::MyString(char* value) {
    this->Value = value;
}

int MyString::getLength() {
    return strlen(this->Value);
}
